sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/FullRestoreOperationHeaders.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The FullRestoreOperationHeaders model. */
+@Fluent
+public final class FullRestoreOperationHeaders {
```

Comment:
Why are headers classes (I saw one above) public? The user shouldn't have to worry about those.

Reply:
Please refer to my comments above on the matter.